.github/BOTMETA.yml

```diff
@@ -213,31 +97,17 @@ files:
   $lookups/manifold.py:
     maintainers: galanoff
     labels: manifold
-  $lookups/nios:
-    maintainers: $team_networking sganesh-infoblox
-    labels: infoblox networking
-  $lookups/passwordstore.py:
-  $lookups/rabbitmq.py:
-    maintainers: Im0
-    labels: rabbitmq
-  $lookups/redis.py:
-  $lookups/shelvefile.py:
-  $module_utils/:
-    labels: module_utils
-  $module_utils/alicloud_ecs.py:
-  $module_utils/cloud.py:
   $module_utils/cloudscale.py:
     maintainers: $team_cloudscale
     labels: cloudscale
+  $module_utils/cloudstack.py:
```

Comment:
I think the cloudstack stuff was moved to https://github.com/ngine-io/ansible-collection-cloudstack